src/scripts/origins.js

```diff
@@ -138,6 +138,10 @@ export default {
     url: '*://*.esa.io/*',
     name: 'Esa.io'
   },
+  EspoCRM: {
+    url: '',
```

Comment:
As there is also a cloud version of EspoCRM available, we can define a url of `*://*.espocloud.eu/*'`